Packs/ctf01/IncidentFields/incidentfield-CTF01.json

```diff
@@ -0,0 +1,29 @@
+{
+    "id": "incident_ctf01",
+    "version": -1,
+    "modified": "2023-08-28T07:25:15.706668791Z",
+    "name": "CTF01",
+    "ownerOnly": false,
+    "description": "Part of the CTF pack",
+    "cliName": "ctf01",
+    "type": "timer",
+    "closeForm": false,
+    "editForm": false,
+    "required": false,
+    "neverSetAsRequired": false,
+    "isReadOnly": true,
+    "useAsKpi": false,
+    "locked": false,
+    "system": false,
+    "content": true,
+    "group": 0,
+    "hidden": true,
+    "openEnded": false,
+    "associatedToAll": true,
```

Comment:
let's associate to an incident type, shouldn be associate to all.

Reply:
we don't have an inc type for `ctf01` also we are not mentioning in the lab guide to use it when a user's starting the CTF. Would you like me to create one just in case?
![image](https://github.com/demisto/content/assets/88268646/d2fcfdae-eab3-454f-b9da-4b32e5dc1782)